Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -113,7 +113,14 @@ extension NSObject {
                 } else {
                     traitSpecifiers.append(strings.switchButtonOffTraitName)
                 }
-
+            } else if let accessibilityValue = self.accessibilityValue {
```

Comment:
I need to test more to ensure that this condition doesn't break the handling for the accessibility traits containing `.button` below.